tests/foreman/installer/test_installer.py

```diff
@@ -1627,19 +1627,10 @@ def test_positive_check_installer_hammer_ping(target_sat):
     :CaseLevel: System
     """
     # check status reported by hammer ping command
-    result = target_sat.execute('hammer ping')
-    test_result = {}
-    service = None
-    for line in result.stdout.strip().replace(' ', '').split('\n'):
-        if line.split(':')[0] not in ('Status', 'ServerResponse', 'message'):
-            service = line.split(':')[0]
-            test_result[service] = {}
-        else:
-            key, value = line.split(":", 1)
-            test_result[service][key] = value
-
-    not_ok = {svc: result for svc, result in test_result.items() if result['Status'] != 'ok'}
-    assert not not_ok
+    result = target_sat.cli.Base.ping()
```

Comment:
I would prefer the original code `target_sat.execute('hammer ping')`. It allows to investigate stdout, stderr and return code.
It seems that `cli.Base.ping()` simply returns only stdout on success or raises _CLIReturnCodeError_ on failure (ping returns non-zero value if any service returns status FAIL). Further assertions are redundant.